### PR TITLE
:card_file_box: Reference

### PR DESCRIPTION
**Pull request by Anny85-code**
### In this milestone I;
- [x] Created a food model (has_many) relationship with inventory.
- [x] Created a Inventory (has_many) relationship with inventoryfood

### CODE REVIEW

File: db/migrate/20220517200900_add_food_ref_inventory.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class AddFoodRefInventory < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :inventories, :food
+  end
+end
```
Comment (jaflih) [inline db/migrate/20220517200900_add_food_ref_inventory.rb:5]: You don't need to create a new migration, just you have to update your model like this :

Capture d’écran 2022-05-18 à 05 48 03

So in the User model you have to add : 

```
has_many :foods, dependent: :destroy
has_many :inventories, dependent: :destroy
```

In the model Inventory : 
`has_many :inventoryfoods, dependent: :destroy`

In the model Food : 
`has_many :inventoryfoods, dependent: :destroy`

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @Anny85-code 

Great work on your project so far, you have really done well!
However there is a little issue you need to fix, I would suggest adding the has_many relation in your models
Happy coding!

**Comment by Anny85-code** [comment]
Hi @jaflih 👋 ,
Thank you for your review, However, I was hoping you could drop some comments to highlights the changes you requested.
Thanks.

**Comment by Anny85-code** [comment]
Thanks for your review. Require suggested changes implemented.

File: app/models/inventory.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
 class Inventory < ApplicationRecord
   belongs_to :user
+  has_many :inventoryfoods, dependent: :destroy
```
Comment (jaflih) [inline app/models/inventory.rb:3]: Change this line by : 
`  has_many :inventory_foods, dependent: :destroy
`

File: app/models/food.rb [hunk 3]
```diff
@@ -1,4 +1,5 @@
 class Food < ApplicationRecord
   belongs_to :user
   has_many :recipe_foods, dependent: :destroy
+  has_many :inventoryfoods, dependent: :destroy
```
Comment (jaflih) [inline app/models/food.rb:4]: Change this line by : 
`  has_many :inventory_foods, dependent: :destroy
`

**Review by jaflih — APPROVED** [review]
Hi @Anny85-code 👋 ,

Please, update your code by the code in the comments, and you can merge it.
Your project is complete, there is nothing else to say than to merge.

**Pull request merged**